flocker/provision/_install.py

```diff
@@ -282,6 +275,57 @@ def task_create_flocker_pool_file():
     ])
 
 
+def task_install_zfs(distribution, variants=set()):
+    commands = []
+    if distribution == 'ubuntu-14.04':
```

Comment:
There is probably a better organization of this code (this file is getting a little unweildly). But given the rest of the code follow this style, I don' think there is currently a benefit in splitting this into three functions. I'm open to suggestions (But I think it is probably worth a separate issue)